examples/plugins/src/file-size.plugin.ts

```diff
@@ -1,18 +1,18 @@
-import { readdir, stat } from 'node:fs/promises';
+import { stat } from 'node:fs/promises';
 import { basename, join } from 'node:path';
-import {
-  formatBytes,
-  pluralize,
-  toUnixPath,
-} from '../../../dist/packages/utils';
+import { formatBytes, toUnixPath } from '../../../dist/packages/utils';
 import {
   AuditOutput,
   AuditOutputs,
   CategoryRef,
   Issue,
-  IssueSeverity,
   PluginConfig,
 } from '../../../packages/models/src';
+import {
+  crawlFileSystem,
+  factorOf,
+  pluralizeToken,
+} from '../../../packages/utils/src';
```

Comment:
```suggestion
} from '@code-pushup/models';
import {
  crawlFileSystem,
  factorOf,
  pluralizeToken,
} from '@code-pushup/utils';
```

The `@nx/enforce-module-boundaries` would've caught this. Maybe better to add an `eslint-disable-next-line` comment where needed, instead of turning the rule off for the whole project?